README.md

```diff
@@ -12,7 +12,7 @@ Please refer to the CONTRIBUTING.md guide for more information. If anything is m
 
 ## Database
 
-All the details on the database schema can be found on the [wiki](https://github.com/tosdr/phoenix/wiki/database).
+All the details on the database schema can be found on the [wiki](https://github.com/tosdr/edit-tosdr-org/wiki/database).
```

Comment:
So apparently you did a rename - but the URL of the current page for me says `edit.tosdr.org`, not `edit-tosdr-org`...

Reply:
Ah good catch!